src/z3c/insist/insist.py

```diff
@@ -358,10 +359,11 @@ def getConfigPath(self):
     def getConfigFilename(self):
         return self.section + '.ini'
 
-    def getIncludes(self, cfgstr):
-        basePath = self.getConfigPath()
+    def getIncludes(self, cfgstr, configPath):
```

Comment:
This breaks public interface. Maybe I should make it optional and use old method if user didn't specify configPath?

Reply:
I think breaking the interface is OK as long as we bump at least the minor the version and document the change in the changelog.